### PR TITLE
Fix an issue with start and end timezones

### PR DESCRIPTION
**Pull request by rgazelot**

### CODE REVIEW

**Comment by rgazelot** [comment]
@Taluu Open for review

**Comment by krichprollsch** [comment]
:+1: :clap:

File: src/Model/Event.php [hunk 1]
```diff
@@ -271,24 +271,25 @@ public static function hydrate(array $data, Calendar $calendar = null)
...
-        if (isset($data['start']) && isset($data['start']['dateTime'])) {
-            $event->start = new Datetime($data['start']['dateTime']);
-        }
-
-        if (isset($data['end']) && isset($data['end']['dateTime'])) {
-            $event->end = new Datetime($data['end']['dateTime']);
-        }
-
         $windowsTimezone = new WindowsTimezone();
+        $endTimeZone = $startTimeZone = 'UTC';
```
Comment (Taluu) [inline src/Model/Event.php]: shouldn't this be `new DateTimeZone('UTC')` ?

**Comment by Taluu** [comment]
Review done. The thing about the datetimezone is not mandatory though

File: src/Model/Event.php (hunk 1 above)
Comment (rgazelot) [inline src/Model/Event.php]: Yes indeed, good catch

**Pull request merged**